pkg/apis/compliance/v1alpha1/compliancesuite_types.go

```diff
@@ -4,6 +4,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ComplianceSuiteIndicatorLabel indicates that an object (normally
```

Comment:
Typo? Should the comment say `SuiteLabel` ?

Reply:
It should.

Reply:
Fixed.